### ISSUE TITLE
Glossary: "tile" term is prone to confusion

### ISSUE THREAD

**Issue by remtav** [microsoft/torchgeo#516]
The [glossary](https://github.com/microsoft/torchgeo/blob/main/docs/user/glossary.rst) you've put up is very useful to try and set a standard for geospatial deep-learning related terms. However in the current version, the term "tile" is prone to confusion.

In torchgeo's glossary:
[tile](https://torchgeo.readthedocs.io/en/latest/user/glossary.html#term-tile) = A single image file taken by a remote sensor like a satellite.
[tiling](https://torchgeo.readthedocs.io/en/latest/user/glossary.html#term-tiling) = Splitting a [tile](https://torchgeo.readthedocs.io/en/latest/user/glossary.html#term-tile) into [patches](https://torchgeo.readthedocs.io/en/latest/user/glossary.html#term-patch). This is the reverse operation of [stitching](https://torchgeo.readthedocs.io/en/latest/user/glossary.html#term-stitching).
Therefore, "tiling" isn't "the process of creating tiles from imagery" but, rather "the process of creating chips/patches from tiles", which is confusing. For example, you'd think that a "clipping" process *creates* "clips", rather than cut up a "clip" (entire image) into smaller chips/patches/pieces.

In the [Cloud Optimized Geotiff spec](https://github.com/cogeotiff/cog-spec/blob/master/spec.md), the term "tile" refers to the image internal blocks (e.g. 512 x 512 px at various scales) which allow to compartmentalize the image to allow range requests.  Consequently, a COG "tile" seems to be more similar to the term "patch" or "chip" from Torchgeo's standpoint.

I'd simply suggest using "scene" or "image" instead of "tile". "tile" could become a synonym of "chip" or "patch".,

The term "tiling", however, seems to be used by other libraries such as [solaris](https://solaris.readthedocs.io/en/latest/tutorials/notebooks/api_tiling_tutorial.html). This term could remain in my opinion.

As a final note, there seems to be a need for a EO-DL synset as a way to reference all these specific terms and synonyms. Your glossary is a good starting point :)

**Comment by adamjstewart** [comment]
Yes, completely agree, the current terminology is quite confusing. I also see "tile" used to refer to WRS row/path "tiles". I'm a bit afraid of trying to standardize our own terminology if the remote sensing community hasn't yet standardized their own terminology. 

I think it also depends on the scale you're working at. If a "tile" is just a smaller section of an image, you could think of patches/chips as tiles, or you could think of each file as a separate tile of the entire Earth.

Are there any popular remote sensing textbooks that standardize this terminology? We should try to stick to any existing established standards whenever possible. I certainly don't want to use a different definition from them and further exacerbate the issue.

TL;DR: I'm fine with renaming "tile" to "scene" in our glossary, but we still refer to "tiles" in hundreds of places in the library. Users will still want to know what we mean by "tile", so we need to have that defined somewhere, even if it's just to say it could mean multiple things.
